packages/curve-ui-kit/src/shared/ui/LargeTokenInput.tsx

```diff
@@ -242,6 +272,10 @@ export const LargeTokenInput = <T extends Amount>({
   const showSlider = maxBalance && maxBalance.showSlider === true
   const showWalletBalance = maxBalance && maxBalance.showBalance !== false
 
+  const chips =
+    maxBalance?.chips === 'max' ? CHIPS_MAX : maxBalance?.chips === 'range' ? CHIPS_RANGE : maxBalance?.chips
```

Comment:
nitpick: a record would be a bit easier to follow here
```suggestion
    {max: CHIPS_MAX, range: CHIPS_RANGE}[maxBalance?.chips] ?? maxBalance?.chips
```